src/app/report/detailReport.tsx

```
@@ -0,0 +1,100 @@
+          <Spacing size={20} />
+          <form.AppField
+            name="attachments"
+            children={(field) => (
+              <field.AttachmentField title="첨부 파일" buttonTitle="파일 +" />
+            )}
+          />
+          <Spacing size={20} />
+          <form.AppField
+            name="address"
+            children={(field) => (
+              <field.PositionSelectField
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)